test/e2e/network/service.go

```diff
@@ -328,7 +328,11 @@ func verifyServeHostnameServiceUp(c clientset.Interface, ns string, expectedPods
 
 	// verify service from pod
 	cmdFunc := func(podName string) string {
-		wgetCmd := "wget -q -T 1 -O -"
+		wgetCmd := "wget -q -O -"
+		// Command 'wget' in Windows image may not support option 'T'
```

Comment:
may not?

Is this test using an actual wget binary or relying on the powershell wget powershell alias that is actually just an alias for Invoke-WebRequest?

It looks like this test is using anghost which uses powershell as the default so i _think_ this is just using the powershell alias - which also doesn't support -q.

Reply:
It is using wget binary:

```
STEP: verifying service has 3 reachable backends
Feb  6 23:42:16.428: INFO: Executing cmd "for i in $(seq 1 150); do wget -q -O - http://100.70.143.81:80 2>&1 || true; echo; done" in pod services-2563/verify-service-up-host-exec-pod
```
here are args of command wget:

```
/ # wget -h
wget: unknown option -- h                                                       
BusyBox v1.29.0.git (2018-10-29 11:20:48 PDT) multi-call binary                 
                                                                                
Usage: wget [-c|--continue] [--spider] [-q|--quiet] [-O|--output-document FILE] 
        [--header 'header: value'] [-Y|--proxy on/off] [-P DIR]                 
        [-S|--server-response] [-U|--user-agent AGENT] URL...                   
                                                                                
Retrieve files via HTTP or FTP                                                  
                                                                                
        --spider        Only check URL existence: $? is 0 if exists             
        -c              Continue retrieval of aborted transfer                  
        -q              Quiet                                                   
        -P DIR          Save to DIR (default .)                                 
        -S              Show server response                                    
        -O FILE         Save to FILE ('-' for stdout)                           
        -U STR          Use STR for User-Agent header                           
        -Y on/off       Use proxy 
```

It seems feature FEATURE_WGET_TIMEOUT is not enabled when busybox-1.29 was built for windows.

Reply:
Thanks!